packages/utilities/src/EventGroup.ts

```diff
@@ -128,14 +129,15 @@ export class EventGroup {
     return !!declaredEvents && !!declaredEvents[eventName];
   }
 
-  public static stopPropagation(event: any) {
+  public static stopPropagation(event: any): void {
     if (event.stopPropagation) {
       event.stopPropagation();
     } else { // IE8
       event.cancelBubble = true;
     }
   }
 
+  // tslint:disable-next-line:typedef
```

Comment:
I disabled the typedef rule here -- I had trouble with setting the return type to `boolean` because `target.addEventListener` could not be converted to boolean. Is that line simply checking if the property exists? Should it be `target.addEventListener !== undefined`? addEventListener itself returns void.

Reply:
I think `typeof target.addEventListener === 'function'` or `!!target.addEventListener` should suffice here.